charts/karpenter/crds/karpenter.k8s.aws_awsnodetemplates.yaml

```diff
@@ -42,6 +42,11 @@ spec:
               amiFamily:
                 description: AMIFamily is the AMI family that instances use.
                 type: string
+              amiSelector:
+                additionalProperties:
```

Comment:
should this still have `additionalProperties`?

Reply:
Ooh, this is auto-generated, I wonder what happened here. Lemme fix.

Reply:
I deleted and recreated the CRD and I think this is just part of the default CRD spec. You'll see it exists for subnetSelector and securityGroupSelector too

Reply:
oh weird! ok 👍